cli/command/utils.go

```diff
@@ -77,7 +80,9 @@ func PrettyPrint(i any) string {
 // the user input 'y' or 'Y' it returns true other false.  If no
 // message is provided "Are you sure you want to proceed? [y/N] "
 // will be used instead.
-func PromptForConfirmation(ins io.Reader, outs io.Writer, message string) bool {
+// If the user terminates the CLI with SIGINT or SIGTERM while
+// the prompt is active, the prompt will return false.
+func PromptForConfirmation(ctx context.Context, ins io.Reader, outs io.Writer, message string) bool {
```

Comment:
Generally speaking, if the user inputs something that's not y/Y/n/N or CTRL-C, do we want to assume the answer was negative, or re-prompt the user?

Reply:
I honestly think it makes sense to just fallback to accepting the value as negative since sensitive commands are usually run with a prompt. Similar to always deny in security. 

Re-prompting can be discussed, but I don't think it adds much value since an incorrect entry assumes you made a mistake (or fell asleep on your keyboard? :) ), so exiting the program makes total sense. Most of the time the command can quickly be rerun with the same command. 

Maybe with otel we might see more insight into failure rates of such prompts and then discuss this further?